src/parquet/column/column-io-benchmark.cc

```diff
@@ -57,17 +57,19 @@ void SetBytesProcessed(::benchmark::State& state, Repetition::type repetition) {
 
 template <Repetition::type repetition>
 static void BM_WriteInt64Column(::benchmark::State& state) {
-  format::ColumnChunk metadata;
+  format::ColumnChunk thrift_metadata;
   std::vector<int64_t> values(state.range_x(), 128);
   std::vector<int16_t> definition_levels(state.range_x(), 1);
   std::vector<int16_t> repetition_levels(state.range_x(), 0);
   std::shared_ptr<ColumnDescriptor> schema = Int64Schema(repetition);
   std::shared_ptr<parquet::WriterProperties> properties = default_writer_properties();
+  auto metadata = parquet::ColumnChunkMetaDataBuilder::Make(
```

Comment:
No need for the `parquet::` prefix here, the benchmarks are also in the same namespace.